packs/hashicups/templates/hashicups.nomad.tpl

```diff
@@ -0,0 +1,171 @@
+job "hashicups" {
+  type   = "service"
+  region = "[[ .hashicups.region ]]"
+  datacenters = [ [[ range $idx, $dc := .hashicups.datacenters ]][[if $idx]],[[end]][[ $dc | quote ]][[ end ]] ]
```

Comment:
You can just use `datacenters = [[ .my.datacenters  | toStringList ]]` here.

Also if you want to use `.my` instead of `.hashicups` thats a thing now, but is not necessary.